.github/workflows/update_licenses.yml

```
@@ -0,0 +1,61 @@
+name: "Update licenses"
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 1 * *" # run once a month
+
+
+jobs:
+  update-licenses:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
+
+      - name: Update licenses list
+        run: |
+          python src/poetry/core/spdx/helpers.py
+
+      - name: Detect changes
+        id: changes
+        run: |
+           echo "count=$(git status --porcelain=v1 2>/dev/null | wc -l) >> $GITHUB_OUTPUT"
+
+      - name: Commit & push changes
+        # Only push if changes exist
+        if: steps.changes.outputs.count > 0
+        run: |
+          git config user.name poetry-cron-bot
+          git config user.email poetry-cron-bot@github.com
```

```suggestion
          git config user.name github-actions[bot]
          git config user.email 41898282+github-actions[bot]@users.noreply.github.com
```

This should make it look like @github-actions did it in the commit log. I've seen this used elsewhere and I've used it myself [here](https://github.com/colindean/fillertext/blob/2922a721878871626cb1c1f6ec8fbb449a51e803/.github/workflows/release.yml#L34-L35).